index.d.ts

```diff
@@ -0,0 +1,223 @@
+import { Readable } from "stream";
```

Comment:
Actually, I realized that `Readable` isn't used anywhere in the code, I think it should be `ReadableStream`.
```suggestion
/// <reference lib="dom" />
import { ReadableStream } from "stream";
```

Reply:
(Other way around actually -- import was correct, usage in code should have been `Readable`)